doc/Language/typesystem.rakudoc

```diff
@@ -648,7 +648,7 @@ Default parameters can be provided.
 Roles can be used as type constraints wherever a type is expected. If a role is
 mixed in with C<does> or C<but>, its type-object is added to the type-object
 list of the object in question. If a role is used instead of a class (using
-auto-punning), the auto-generated class' type-object, of the same name as the
+auto-punning), the auto-generated class's type-object, of the same name as the
```

Comment:
class’ is correct based on the wikipedia style guide as referenced under writing-docs.

Reply:
possessives links to https://en.wikipedia.org/wiki/Apostrophe, includes this text:

> Plural nouns already ending in ⟨s⟩ take only an apostrophe after the pre-existing ⟨s⟩ to form the possessive: e.g., "three cats' toys".